icubam/predicu/__init__.py

```diff
@@ -1 +0,0 @@
-from .data import load_all_data, load_data_file, load_pre_icubam_data
```

Comment:
For `__init__` we can rather put these in,
```py
__all__ = ['load_all_data', 'load_data_file', 'load_pre_icubam_data']
```
which will fix linting. I'm not really sure we want to expose these at the module level, but in any case it's not a linting issue.

Reply:
I'll just revert that change then, seeing as the CI flake8 configuration skips predicu anyway